guides/adr/0011-tda-courses.md

```diff
@@ -31,6 +31,31 @@ courses seamlessly. Furthermore, the details course tab requires enhancements to
 automatically set degrees as not required for TDA courses.
 Lastly, there's a need for a filtering mechanism in the Find module to streamline course discovery.
 
+## Options
+
+### Update `Course#program_type` enum to include a new value: 'tda'
```

Comment:
If we go down this route? Can we call the value `teacher_degree_apprenticeship` instead of `tda`?